client/ayon_blender/plugins/publish/extract_fbx_animation.py

```diff
@@ -132,13 +133,35 @@ def process(self, instance):
         max_frame = min(starting_frames)
         min_frame = max(ending_frames)
 
-        # We bake the copy of the current action for each object
-        bpy_extras.anim_utils.bake_action_objects(
-            object_action_pairs,
-            frames=range(int(min_frame), int(max_frame)),
-            do_object=False,
-            do_clean=False
-        )
+        ver_major, ver_min, _ = get_blender_version()
+        if ver_major >= 4 and ver_min >= 1:
```

Comment:
This unfortunately wouldn't work in Blender 5.0 would be released because it wouldn't pass the minor check.
```suggestion
        blender_version = get_blender_version()
        if blender_version >= (4, 1, 0):
```

E.g.

```python
>>> (4, 1, 0) >= (4, 0, 0)
True
>>> (4, 0, 0) >= (4, 1, 0)
False
>>> (3, 9, 2) >= (4, 1, 0)
False
>>> (4, 2, 0) >= (4, 1, 0)
True
>>> (5, 0, 0) >= (4, 1, 0)
True
```